oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -273,4 +280,73 @@ private static long getFileDescriptors(int index) {
         }
         return 0L;
     }
+
+    /**
+     * Get logical volume groups on this machine
+     *
+     * Instantiates a list of {@link VolumeGroup} objects,
+     * representing a storage pool, device, partition, volume, concrete file system
+     * or other implementation specific means of file storage.
+     *
+     * @return A list of {@link VolumeGroup} objects or an empty
+     *      *         array if none are present.
+     */
+
+    public List<VolumeGroup> getLogicalVolumeGroups() {
```

Comment:
Per discussion on #1578, I think this might be better on the Hardware Abstraction Layer.  Interface implementation should be `default` returning `Collections.emptyList()`.